### ISSUE TITLE
[Question] Windows phone support

### ISSUE THREAD

**Issue by radiovisual** [SamVerschueren/mobisplash#1]
Is there any planned support for windows phones?

| Icon name | 240% | 140% | 100% |
| --- | --- | --- | --- |
| Splash Screen | 1152×1920 | 672×1120 | 480×800 |

**Note:** The chart above is just a sample, and isn't 100% complete.

**Comment by SamVerschueren** [comment]
Yes sure, would like to have some help with this though, not very familiair with WP development.

**Comment by radiovisual** [comment]
No problem. I can help out with this. The sizes for Windows phone splash screens are pretty straight-forward.
